discovery-kubernetes-api/src/main/scala/akka/discovery/kubernetes/KubernetesApiServiceDiscovery.scala

```diff
@@ -63,9 +64,9 @@ object KubernetesApiServiceDiscovery {
           } yield Some(port.containerPort)
       }
     } yield {
-      val host = s"${ip.replace('.', '-')}.$podNamespace.pod.$podDomain"
+      val hostOrIp = if (rawIp) ip else s"${ip.replace('.', '-')}.$podNamespace.pod.$podDomain"
```

Comment:
does this need `[]` brackets when it's an IPv6 IP?

Reply:
I don`t know, but according to the request for this change, this was not mentioned.